runtime-sdk/modules/evm/src/lib.rs

```diff
@@ -116,6 +119,37 @@ pub trait Config: 'static {
             EVM_CONFIG.get_or_init(EVMConfig::shanghai)
         }
     }
+
+    /// Returns the whitelisted EVM storage slots.
+    ///
```

Comment:
Should this be defined outside of the function and outside of the config, removing the need for this function?

Reply:
I think you are right, maybe we don't need this to be configurable. I guess doing a new release of the runtime SDK, if/when we need to update this, is fine? 

wdyt @kostko ?

Reply:
Yeah it should be fine to not make this configurable.